### PR TITLE
Create On-point Rotation

### PR DESCRIPTION
**Pull request by cvalarida**
I'm not sure about the naming, but this outlines the basic structure we discussed in https://github.com/department-of-veterans-affairs/va.gov-team/issues/410.

What do y'all think?

### CODE REVIEW

File: Platform/Teams/Triage/On-point Rotation.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## Responsibilities
+When we get an alert in the Slack channel, the on-point engineer will triage the issue by doing any of the following:
+
+- Tweak the alert rule
+- Make the code fix
+- Mark as "ignored" or "resolved" in Sentry
+- Create a GitHub issue for the Sentry issue
+
+Regardless of the action taken, the engineer will state what they did in a thread.
```
Comment (omgitsbillryan) [inline Platform/Teams/Triage/On-point Rotation.md]:
```suggestion
Stating the action taken by way of Slack thread and/or Sentry Issue comment is encouraged. This helps indicate to others that this Sentry Alert has been looked at.
```

**Review by omgitsbillryan — APPROVED** [review]
LGTM 👍

File: Platform/Teams/Triage/On-point Rotation.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- Create a GitHub issue for the Sentry issue
+
+Stating the action taken by way of Slack thread and/or Sentry Issue comment is encouraged. This helps indicate to others that this Sentry Alert has been looked at.
+
+### Getting help
+If the on-point engineer doesn't know how to effectively triage the issue (for example, if they don't know how to assess the user impact), they're encouraged to reach out to the other Triage engineers for guidance.
+
+
+## Schedule
+The on-point responsibilities are rotated among the engineers of the Triage team on a weekly basis.
```
Comment (kfrz) [inline Platform/Teams/Triage/On-point Rotation.md]:
```suggestion
The on-point responsibilities are rotated among the engineers of the Triage team on a weekly basis. Weeks are defined as beginning on Monday and encompass only business hours.
```

File: Platform/Teams/Triage/On-point Rotation.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+## Purpose
+Not every bug can be fixed, but every bug _can_ be triaged.
+
+Triaging all the issues that come in is a team effort, but a little leadership goes a long way. Each week, a Triage team engineer will take point on triaging the issues from our intake sources.
+
+We'll give this process a test run by triaging Sentry alerts in the `#vetsgov-sentry-alerts` Slack channel.
+
+## Responsibilities
+When we get an alert in the Slack channel, the on-point engineer will triage the issue by doing any of the following:
```
Comment (kfrz) [inline Platform/Teams/Triage/On-point Rotation.md]:
```suggestion
When we get an alert in the `#vetsgov-sentry-alerts` Slack channel, the on-point engineer will triage the issue by doing any of the following:
```

**Review by kfrz — APPROVED** [review]
LGTM, minor suggestions, take em or leave em

**Pull request merged**